package.json

```diff
@@ -120,5 +120,8 @@
   "volta": {
     "node": "20.11.0",
     "npm": "9.6.3"
+  },
+  "dependencies": {
+    "execa": "^8.0.1"
```

Comment:
Should this dependency be removed as well then, now that the script is gone?

Reply:
done!